wled00/data/simple.css

```diff
@@ -180,15 +174,15 @@ button:hover {
 .tab button {
 	background-color: transparent;
 	float: left;
-	border: none;
-	transition: color 0.3s, background-color 0.3s;
+	border: 0;
+	transition: color .3s, background-color .3s;
 	font-size: 17px;
 	color: var(--c-c);
 	min-width: 44px;
 }
 
 .top button {
-	padding: var(--tbp);
+	padding: 14px 8px 10px;
```

Comment:
This alone is dealbreaker. Please revert the change.

Reply:
Happy to revert, but can you tell me the reason for this decision?